registry.json

```diff
@@ -1787,5 +1787,20 @@
     "telegram":"manizzler",
     "operatorName":"manizzler",
     "github":"manizzle"
+  },
+  {
+    "ticker":"VLHLA",
+    "poolId":"29889ecc8151e5a93fcd321cd905eee0f50c8364273550205058bfe8",
+    "social":{
+      "facebook":"",
+      "twitter":"https://twitter.com/valhallapool",
+      "website":"http://valhallapool.net",
+      "reddit":"",
+      "instagram":"",
+      "youtube":""
+    },
+    "telegram":"valhallapool",
```

Comment:
Can't find this Telegram username

Reply:
Resolved with commit dba82500c4dd7dece29df2cc7c7813fc24a6a0ac